store/sqlstore/user_store.go

```diff
@@ -319,23 +320,36 @@ func (us SqlUserStore) UpdateMfaActive(userId string, active bool) store.StoreCh
 	})
 }
 
+// usersQuery is a starting point for all queries that return one or more Users.
+var usersQuery *querybuilder.Query = querybuilder.
+	Select("u.*").
```

Comment:
In my bots PR, I'll add a new `Select` and `LeftJoin` to pull out the column across all queries.